Gems/Terrain/Code/CMakeLists.txt

```diff
@@ -78,8 +76,8 @@ if(PAL_TRAIT_BUILD_HOST_TOOLS)
     )
 
     # the above module is for use in dev tool situations
-    ly_create_alias(NAME Terrain.Builders NAMESPACE Gem TARGETS Gem::Terrain.Editor)
-    ly_create_alias(NAME Terrain.Tools    NAMESPACE Gem TARGETS Gem::Terrain.Editor)
+    ly_create_alias(NAME Terrain.Builders NAMESPACE Gem TARGETS Gem::Terrain.Editor Gem::SurfaceData.Editor Gem::GradientSignal.Editor)
+    ly_create_alias(NAME Terrain.Tools    NAMESPACE Gem TARGETS Gem::Terrain.Editor Gem::SurfaceData.Editor Gem::GradientSignal.Editor)
```

Comment:
It may be safer to assign GradientSignal.Builders and GradientSignal.Tools here (assuming those variants exist).

This would allow GradientSignal itself to decide to split to using a specific module for builders instead of reusing the editor module in builders at a later date, without having to go here and update htis.